### PR TITLE
Rebased the LSP4ij integration branch with the new branch lsp4ij-market-0.0.2-integration

### PR DESCRIPTION
**Pull request by aparnamichael**
Rebased the LSP4ij integration branch with the new branch lsp4ij-market-0.0.2-integration

### CODE REVIEW

File: src/main/resources/META-INF/lsp.xml [hunk 1]
```diff
@@ -9,11 +9,11 @@
   -->
 
 <idea-plugin>
-    <extensions defaultExtensionNs="open-liberty.intellij">
+    <extensions defaultExtensionNs="com.redhat.devtools.lsp4ij">
         <!-- MicroProfile LS -->
         <server id="lsp4mp"
                 name="MicroProfile"
-                factoryClass="io.openliberty.tools.intellij.lsp4mp.lsp.MicroProfileLanguageServerFactory"
+                factoryClass="io.openliberty.tools.intellij.lsp4mp.lsp.MircoProfileLanguageServerFactory"
```
Comment (turkeylurkey) [inline src/main/resources/META-INF/lsp.xml:16]: This bug was fixed in other branches, please leave it MicroProfileLanguageServerFactory.

**Review by turkeylurkey — CHANGES_REQUESTED** [review]

File: src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp/MircoProfileLanguageServerFactory.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
```
[the remark this answers is not in the document]
Comment (aparnamichael) [reply src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp/MircoProfileLanguageServerFactory.java]: @turkeylurkey Addressed the review comments.

**Review by turkeylurkey — APPROVED** [review]

File: src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/hover/PropertiesHoverParticipant.java [hunk 3]
```diff
@@ -211,34 +222,30 @@ protected boolean isAdaptableFor(PsiElement hoverElement) {
...
 	 * @return the config property information for the given property key
 	 */
 	private static List<MicroProfileConfigPropertyInformation> getConfigPropertyInformation(String propertyKey,
-			PsiAnnotation annotation, String defaultValueAnnotationMemberName, PsiFile typeRoot,
-			PsiMicroProfileProject project, IPsiUtils utils) {
+																							PsiAnnotation annotation,
+																							String defaultValue,
+																							PsiFile typeRoot,
+																							PsiMicroProfileProject project,
+																							IPsiUtils utils) {
```
Comment (TrevCraw) [inline src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/hover/PropertiesHoverParticipant.java:241]: what happened with the formatting here?

Comment (aparnamichael) [reply src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/hover/PropertiesHoverParticipant.java]: When I replaced this file with Quarkus's file  this custom lines breaks came.

**Review by TrevCraw — APPROVED** [review]

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**